fuzzers/libfuzzer_libpng_launcher/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
```

Comment:
Why a shell script if we have the makefile?

Reply:
yes, I should make Makefile for those fuzzers that are missing Makefile and remove run.sh